tests/installation/partitioning_raid.pm

```diff
@@ -158,6 +164,97 @@ sub set_lvm {
     wait_screen_change { send_key $cmd{finish} };
 }
 
+sub modify_uefi_boot_partition {
+    assert_screen 'partitioning_raid-disk_vdd_with_partitions-selected';
+    # fold the drive tree
+    send_key 'left';
+    assert_screen 'partitioning_raid-hard_disks-unfolded';
```

Comment:
not saying it really should be like this but just thinking loud - can we replace all this send_key + assert_screen with send_key_until_needlematch ?

Reply:
We can do that, but in some places it's about single key press, so I don't see much value in it, except more stability, which is not an issue currently

Reply:
@asmorodskyi I thought about the same. We can make it more explicit by adding a new function `send_key_and_assert_screen left => 'partitioning_raid-hard_disks-unfolded'` but I don't think it will help much. OTOH if by doing this we can encourage more users to do this by default and not try to call multiple send_key in a row without checking results in between then maybe it would help

Reply:
ok